src/v3/src/components/Widget/index.tsx

```diff
@@ -282,6 +283,8 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
       loading,
       setLoading,
       setWidgetRendered,
+      userIdentifier,
+      setUserIdentifier,
```

Comment:
We need to make sure this value is cleared in the case that a user clears the username field (should go back to `null`/`undefined` OR we need to check for empty strings)

Also if a user backs out of a flow by clicking the back button in the widget and the widget shows a blank username field again, we need to ensure the value is cleared as well.

Finally, what happens if the username is prefilled via widgetConfig or rememberMe functionality?

Reply:
if the username is prefilled via widgetConfig or rememberMe it will still work properly since it sets the `userIdentifier` value to whatever is in the username field only after the launch authenticator button is pressed.  The username field data is populated before the authenticator button is pressed.  

Since the `userIdentifier` value is set only after the launch auth button is pressed, we shouldnt have any issue of possibly having the wrong username set in it or the user clearing the field.  So it shouldnt be a problem if someone backs out of the flow, but technically the old username would still be leftover in `userIdentifier` until the launch auth button is pressed again which may be a security concern.  I think a simple solution to this would be to clear the `userIdentifier` as soon as the identify flow loads.

Reply:
> if the username is prefilled via widgetConfig or rememberMe it will still work properly since it sets the userIdentifier value to whatever is in the username field only after the launch authenticator button is pressed. The username field data is populated before the authenticator button is pressed.

👍 

> So it shouldnt be a problem if someone backs out of the flow, but technically the old username would still be leftover in userIdentifier until the launch auth button is pressed again which may be a security concern. I think a simple solution to this would be to clear the userIdentifier as soon as the identify flow loads.

Cool, I think as a part of good hygiene we should clear out these widget context values properly in this case even if it's not affecting the widget behavior in the current use-case.